CRM/Contribute/BAO/Contribution.php

```diff
@@ -5011,6 +5017,8 @@ public static function checkLineItems(&$params) {
     elseif ($totalAmount != $lineItemAmount) {
       throw new API_Exception("Line item total doesn't match with total amount.");
     }
+
+    return TRUE;
```

Comment:
I'm not sure about adding a return value just to support the tests, since we never return FALSE.  If it fails an exception will be thrown, can't we rely on that for the test?

Reply:
>>>  If it fails an exception will be thrown, can't we rely on that for the test?

but then how can we test if supplying correct amounts will result in no error